x-pack/elastic-agent/pkg/agent/application/paths/paths.go

```diff
@@ -29,6 +33,9 @@ func init() {
 	fs.StringVar(&topPath, "path.home", topPath, "Agent root path")
 	fs.StringVar(&configPath, "path.config", configPath, "Config path is the directory Agent looks for its config file")
 	fs.StringVar(&logsPath, "path.logs", logsPath, "Logs path contains Agent log output")
+
+	// create tempdir as it probably don't exists
+	os.MkdirAll(TempDir(), 0750)
```

Comment:
Should this TempDir be wiped on startup to make sure it is always "clean"?

Reply:
we can wipe it but ideally whoever uses it cleans after work is done so we dont have to wait until restart and we free resources